plugins/CoreAdminHome/lang/en.json

```diff
@@ -167,7 +167,7 @@
         "AcceptInvite": "Accept Invite",
         "DeclineInvite": "Decline Invite",
         "UserInviteContent": "You can accept this invitation by clicking the accept button below.",
-        "UserInviteNotes": "If you were not expecting this invitation, you can ignore this email. If %1$s is sending you too many emails, you can block them or report abuse. The invitation automatically expires in %2$s days.",
+        "UserInviteNotes": "<b>Notes:</b> If you were not expecting this invitation, you can ignore this email. If %1$s is sending you too many emails, you can block them or report abuse. The invitation automatically expires in %2$s days.",
```

Comment:
I'd rather tend to simply remove the `Notes`. It does bring any additional value